Documentation/TopLevelObjects/Config.rst

```diff
@@ -503,18 +503,34 @@ Properties of 'config'
         comment in the <head> section of the page. Use this to insert a note
         like that "Programmed by My-Agency".
 
-    ..  confval:: htmlTag.attributes
+    ..  confval:: htmlTag.attributes.[attribute]
         :name: config-htmlTag-attributes
-        :type: array
+        :type: string /
```

Comment:
"string /"? Seems something is missing behind the "/". Do you mean "stdWrap / string" or something like that?